x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,132 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
+          "__template": {
+            "after": "1d",
+            "actions": {
+              "allocate": {
+                "number_of_replicas": 1,
+                "include": {
```

Comment:
@colings86 can you please have a look and double this is a good default config to show users in console?